gateway-service-impl/src/test/resources/expected-test-responses/explore/grouped-time-aggregation-with-group-limit-with-null-value-group.json

```diff
@@ -0,0 +1,45 @@
+{
+  "row": [{
```

Comment:
Whats the query look like for this for gateway?

Reply:
Oh! okey, it will be an internal query. Or would be IN query from GraphQL to gateway too. That is currently not expressible, right?

Reply:
We have all queries in the test
1. GraphQL query to GW
2. GW query to QS and its response
3. GW response to GraphQL